src/therapy/schemas.py

```diff
@@ -239,6 +239,7 @@ class NamespacePrefix(Enum):
     DRUGCENTRAL = "drugcentral"
     DRUGSATFDA_ANDA = "drugsatfda.anda"
     DRUGSATFDA_NDA = "drugsatfda.nda"
+    HCPCS = "hcpcs"  # Healthcare Common Procedure Coding System
```

Comment:
Add corresponding `NAMESPACE_TO_SYSTEM_URI` member. I think I forgot to add the test for this: https://github.com/cancervariants/disease-normalization/blob/main/tests/unit/test_schemas.py